blocks/customers/customers.css

```diff
@@ -0,0 +1,100 @@
+.customers{
+    margin-top: 20px;
```

Comment:
"Margins should not bleed outside their containers" -[here](https://www.joelambert.co.uk/article/bem-guidelines/) | Set this spacing with padding all around the block.

Edit: Looking over the styles below, there are a few collapsing margins and things that move each-other out of alignment as the page size changes. To prevent this, I think the easiest way is to use as few margins set as possible and rely on the higher levels paddings to keep things in place. Explaining this via text is hard... [here's a demo](https://www.loom.com/share/d4c9638c24374c708c45f87306903ae3) and [here's the demo seen in the demo](https://codepen.io/mccambley/pen/GRWzaeg)

Edit: I forgot to mention in the demo, the things that are changed in the Codepen shoooould be beneath a /* new */ comment in the CSS. If following those, just note the `box-sizing: border-box` on `customer__content`. That's a potential breaker.